beer-garden/beer-garden
# #1300 - Include the Systems and Namespaces for the local garden

Closes https://github.com/beer-garden/beer-garden/issues/1300

This PR does the following:

- Populates the Namespaces and Systems for the local garden when making calls to the /api/v1/gardens endpoint.

Test Instructions:
Testing this requires a beergarden with at least one remote garden. Deploy plugins to both gardens.
- Check the Admin->Gardens page, Verify that the Namespaces and Systems isn't 0 for the local garden.
- Run the following command ```curl localhost:2337/api/v1/gardens | jq -S '.'```. Check that there is an entry for ```connection_type: LOCAL``` with systems and namespaces.
